### ISSUE TITLE
Dealing with cost sharing and fare payment for joint tours

### ISSUE THREAD

**Issue by aletzdy** [SANDAG/ABM#140]
The issue #138 started a discussion on whether we need/should divide parking and auto operating costs for shared rides by either party size (aka number_of_participants) or a sharing factor, reflecting that each person sharing the ride sees a lower cost than the full cost.

We recognize that in the special market models, the likelihood of strangers carpooling is small, and a shared ride would be comprised only of the people in the party traveling together. This results in the conclusion that we should not be dividing the auto costs by party size, since the decision-making unit is no longer an "individual", but the party, and they will try to minimize the disutility of their party as a unit. The example would be a family of three who decides to go to a mall together (party_size==3). When making decisions, the party looks at the total auto operating cost and parking costs, and not per head cost, and compares it against other modes' utilities for the whole party. This also results in the conclusion that transit fare needs to be multiplied by party size, reflecting the total fare a traveling party would pay to travel together.

To sum up, we need to make sure all models multiply transit fare by party size. The special market models need NOT divide auto costs by party size, but we leave the resident model treatment of auto cost sharing intact for now, due to the more complex cases of strangers carpooling.

**Comment by aber-sandag** [comment]
https://github.com/SANDAG/ABM/commit/a2d8237027f5dfde19436553aa53c6235ee4facb

**Comment by bhargavasana** [comment]
A summary of the logic team agreed on.
Resident Model
- Apply shared-ride factors to parking costs only and also only for non-joint tours/trips
- For AOC and tolls, do not use shared-ride factors because it has an unintended distance effect
- For joint tours/trips, multiply transit fares by number of participants

Special Market Models
- No shared-ride factors used at all because we assume no carpooling with non-party members
- Multiply transit fares by number of participants

More detailed logic for Microtransit fare treatment (and potentially joint micromobility trips) will be discussed in future.
